js/samples/quickstart-csharp/Controllers/HomeController.cs

```diff
@@ -1,21 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
 using QuickstartSampleWebApp.Models;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Identity.Client;
 
 namespace QuickstartSampleWebApp.Controllers
 {
-	public class HomeController : Controller
-	{
+    public class HomeController : Controller
+    {
 		private readonly string TenantId;     // Azure subscription TenantId
```

Comment:
can reduce indent of this code as well

Reply:
I see it correctly

![image](https://user-images.githubusercontent.com/87383938/157326183-10d4b229-397d-4bb5-a8f5-93307ab7b6d0.png)